soh/soh/Enhancements/randomizer/static_data.h

```diff
@@ -46,7 +45,6 @@ class StaticData {
       static std::vector<RandomizerCheck> GetStaticHintLocations();
       static std::vector<RandomizerCheck> GetPondFishLocations();
       static std::vector<RandomizerCheck> GetOverworldFishLocations();
-      static std::vector<RandomizerCheck> GetOverworldPotLocations();
```

Comment:
Removed this since it's unused, but all these GetOverworld methods should be removed